### ISSUE TITLE
Generate XGboost & Other Ordinal Models on CIS-PD Data

### ISSUE THREAD

**Issue by sieberts** [Sage-Bionetworks/beat-pd#5]

**Comment by atm15** [comment]
I'm gonna call this done for the sake of housekeeping. 3 classification, 2 ordinal, and 2 regression models have been implemented. Happy to implement other ones, but I don't anticipate any significant benefits based on what I've seen so far.

**Comment by sieberts** [comment]
Let's reopen this until you've gotten the individual scores in Synapse. When you close, please provide that location in the comments @atm15. Thanks!

**Comment by atm15** [comment]
My bad! I'm re-running everything now so I save all the numbers that people would want to use to calculate different performance metrics. I'll have them up by EOD.

**Comment by sieberts** [comment]
Per Alex via Slack:
I have uploaded a .zip directory containing spreadsheets showing the results of different models: https://www.synapse.org/#!Synapse:syn20978525. The rows correspond to different folds, the columns correspond to different quantities that are used for scoring metrics: number of samples, MSE, MAE, macro-MSE, macro-MAE, corresponding variances and null model values. If there’s something I’m missing, let me know and I’ll generate it.

The table I generated uses 10 samples as the minimum per class, I’m generating a second zip with 2 as the minimum just for comparison sake
